src/app/commands/poap/Claim.ts

```diff
@@ -1,5 +1,5 @@
 import {
-	CommandContext,
+	CommandContext, CommandOptionType,
```

Comment:
Will you put this on its own line for consistency?

Reply:
yeah idk why the IDE keeps placing it on the same line